src/mom6-tools.plot/sfc.time.plot.py

```diff
@@ -90,7 +90,6 @@ class args:
 
   # load mom6 grid
   grd = MOM6grid(args.grid)
-  grd.area_t = grd.Ah
```

Comment:
@jkbk2004 Why is this been removed?

Reply:
@ShastriPaturi if you want to test with old runs, this is another issue since we are updating diag_table now. I added area_t  in new diag_table so that time_mean directly use area_t instead of using Ah.

Reply:
@ShastriPaturi if you still want to make time_mean plots, you can directly run sfc.time.plot.py as described in readme.